doc/api/child_process.md

```diff
@@ -1094,6 +1095,27 @@ console.log(`Spawned child pid: ${grep.pid}`);
 grep.stdin.end();
 ```
 
+### subprocess.ref()
+<!-- YAML
+added: v0.7.10
+-->
+
+Calling `subprocess.ref()` after making a call to `subprocess.unref()` will
+restore the removed reference count for the child process, allowing the parent
+process to wait for the child process to exit before itself exiting.
```

Comment:
nit: given the context, usage of the word `process` to describe `parent` and `child` can be safely omitted.

Reply:
OK